packages/next-auth/src/react/index.tsx

```diff
@@ -456,6 +462,21 @@ export function SessionProvider(props: SessionProviderProps) {
         : session
         ? "authenticated"
         : "unauthenticated",
+      async update(data) {
+        setLoading(true)
+        const session = await fetchData<Session>(
+          "session",
+          __NEXTAUTH,
+          logger,
+          { req: { body: { csrfToken: await getCsrfToken(), data } } }
+        )
+        setLoading(false)
+        if (session) {
+          setSession(session)
+          broadcast.post({ event: "session", data: { trigger: "getSession" } })
```

Comment:
should it be `updateSession` trigger?

Reply:
We only have `signOut` and `getSession`:

https://github.com/nextauthjs/next-auth/blob/feaeda9e2a2d6768339951fd4daa8aa2cf20ecbf/packages/next-auth/src/client/_utils.ts#L69

Should not really matter, AFACT, we don't really use this value for anything really. From the other window's perspective, it should also not matter what triggered this, it just n eeds to get the updated session.